File: Javascript (NODE JS)/addingTrouble.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

testcases = 0
rl.on('line', (line) => {
    split = line.split(" ")
    if(parseInt(split[0]) + parseInt(split[1]) == parseInt(split[2])){
        console.log("correct!")
    }
    else{
        console.log("wrong!")
    }
});

File: Java/addingTrouble.java
import java.util.Scanner;
public class addingtrouble{
    public static void main(String[]args){
        int a;
        int b;
        int c;
        Scanner sc = new Scanner(System.in);
        String[] s = sc.nextLine().split(" ");
        a = Integer.parseInt(s[0]);
        b = Integer.parseInt(s[1]);
        c = Integer.parseInt(s[2]);
        if(a + b == c){
            System.out.println("correct!");
        }
        else{
            System.out.println("wrong!");
        }
    }
}
